### PR TITLE
Add support for serialization of hashes

### PR DESCRIPTION
**Pull request by phylor**
Currently LWS only supports the serialization of objects. Often it is easier to use hashes though, especially for small data structures.

For example, we'd like to do the following:

```ruby
countries = ISO3166::Country.all.map do |country|
  {
    code: country.alpha2,
    label: country.translation(I18n.locale)
  }
end

render json: countries, serializer: CountrySerializer
```

```ruby
class CountrySerializer < ApplicationSerializer
  attribute :code
  attribute :label
end
```

If you currently try to serialize a hash:

```
NoMethodError:
       undefined method `code' for #<Hash:0x00005578e8dd3450>
```

This PR solves the above error. So `CountrySerializer.new({code: 'DE', label: 'Germany'})` then works fine.

### CODE REVIEW

File: lib/lightweight_serializer/serializer.rb
Comment (rmehner) [inline lib/lightweight_serializer/serializer.rb:274]:
```suggestion
        object.fetch(attribute_config.attr_name)
```

Reasoning: `public_send` will raise a `NoMethodError` if the method does not exist on the object. `hash[:key]` will return `nil` and not raise. In order to have the same behaviour across the board, I'd prefer to have it raise for hashes as well (even though it puts a bit more work on the consumer side, as they have to make sure to always have all the keys set in the hash)

Either way, this should likely be tested too, so it's a well defined behaviour and not accidental behaviour.

**Review by rmehner — COMMENTED** [review]
Neat, exactly what I was missing from this but never came around to implement it myself. One comment though and apparently we should wait for @ans-ioki for a final approval, since he has opinions on hashes 😄

File: lib/lightweight_serializer/serializer.rb
Comment (phylor) [reply lib/lightweight_serializer/serializer.rb]: Good point! Added a test.

Could imagine that this gets annoying at some point for consumers, but we could easily add an option to the serializer not to raise.

**Review by rmehner — APPROVED** [review]

**Comment by klaustopher** [comment]
Nice!

**Pull request merged**
